build/firmware/Common.mk

```diff
@@ -44,7 +50,13 @@ UEFI_KEY_PARAMS = -k $(KEY_ROOT) -r -K dt.dtb
 
 FTPM_FLAGS= \
 	CFG_TEE_TA_LOG_LEVEL=2 \
-	CFG_TA_DEBUG=y \
+	CFG_TA_DEBUG=n \
+	CFG_FTPM_USE_WOLF=n \
+
+AUTHVAR_FLAGS= \
+	CFG_TEE_TA_LOG_LEVEL=2 \
+	CFG_TA_DEBUG=n \
+	CFG_AUTHVARS_USE_WOLF=y \
```

Comment:
CFG_*_USE_WOLF should be defined by the per-board makefiles.

Reply:
Fixed